db/migrate/20210813114631_delete_rolesfrom_user.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteRolesfromUser < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :users, :role, :integer
+  end
+end
```

Comment:
I don't think you understood me. Look at your committed [20210812123938_devise_create_users.rb](https://github.com/Minpeople/monte-cinema/blob/f60b75be4f8c6fe7f7096733dde530af5f05ee7b/db/migrate/20210812123938_devise_create_users.rb) file - `role` column is not defined here but this migration is specified to create whole users table, meaning that at this point the `role` does not exist and running this migrations would fail.